code/__HELPERS/_lists.dm

```diff
@@ -331,6 +331,10 @@
 /proc/sortList(list/L, cmp=/proc/cmp_text_asc)
 	return sortTim(L.Copy(), cmp)
 
+//sortList but with assoc lists
+/proc/sortAssocList(list/L, cmp=/proc/cmp_text_asc)
```

Comment:
Didn't know whether to add an optional argument for assoc lists in the existing `sortList` or to make another proc, so I flipped a coin. It landed on tails.

Reply:
Reasonable choice.